app/controllers/application_controller.rb

```diff
@@ -3,4 +3,8 @@ class ApplicationController < ActionController::API
   rescue_from CanCan::AccessDenied do |exception|
     render json: { error: exception.message }, status: :forbidden
   end
+
+  rescue_from ActionController::UnpermittedParameters do |exception|
+    render json: { error: "You are not authorized to change this parameter." }, status: :forbidden
```

Comment:
This needs to be more generic. They could also be passing a param that doesn't exist or that no one should access.

```suggestion
    render json: { error: "Forbidden" }, status: :forbidden
```

A good rule of thumb with errors: if the error is because someone is not _authorized_, say as little as possible.